src/Microsoft.DotNet.Interactive.Tests/StdioConnectionTests.cs

```diff
@@ -0,0 +1,158 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Reactive.Disposables;
+using System.Text;
+using System.Threading.Tasks;
+using FluentAssertions;
+using Microsoft.DotNet.Interactive.Commands;
+using Microsoft.DotNet.Interactive.Connection;
+using Microsoft.DotNet.Interactive.Events;
+using Microsoft.DotNet.Interactive.Tests.Utility;
+using Microsoft.DotNet.Interactive.Utility;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.DotNet.Interactive.Tests;
+
+public class StdioConnectionTests : KernelConnectionTestsBase<StdioConnectionTestConfiguration>
+{
+    public StdioConnectionTests(ITestOutputHelper output) : base(output)
+    {
+    }
+
+
+    protected override Task<IKernelConnector> CreateConnectorAsync(StdioConnectionTestConfiguration configuration)
+    {
+        var command = new List<string>{configuration.Command};
+        if (configuration.Args?.Length > 0)
+        {
+            command.AddRange(configuration.Args);
+        }
+
+        var connector = new StdIoKernelConnector(command.ToArray(), configuration.WorkingDirectory);
```

Comment:
This constructor should be fixed to differentiate the executable path from the subsequent command line args.